src/main/java/nl/tudelft/cse1110/andy/execution/ExecutionStep.java

```diff
@@ -3,5 +3,5 @@
 import nl.tudelft.cse1110.andy.result.ResultBuilder;
 
 public interface ExecutionStep {
-    void execute(Context ctx, ResultBuilder result);
```

Comment:
Ohhh why is that? Do we really need to change the method signature to indicate that it throws a _generic_ exception?